src/test/scala/ai/privado/auth/AuthHandlerTest.scala

```diff
@@ -0,0 +1,15 @@
+package ai.privado.auth
```

Comment:
Is this file is not needed can we remove this @pandurangpatil (non blocker)

Reply:
I was trying to add unit test for the logical flow, as it was difficult to test the different combinations. However, it was taking a lot of time as had to refactor a lot of original code. So I did the combinations tested manually by replicating different scenarios. But I kept this TODO by removing the code, so we don't forget to add the unit tests. Let's keep it there for now.